web_services/foxx/brightid/initdb.js

```diff
@@ -20,16 +20,17 @@ const collections = {
 };
 
 // deprecated collections should be added to this array after releasing
-// second update to allow 2 last released versions work together 
+// second update to allow 2 last released versions work together
 const deprecated = [
   'removed',
   'newGroups',
   'usersInNewGroups',
 ];
 
 const indexes = [
-  {'collection': 'verifications',  'fields': ['name']},
-  {'collection': 'verifications',  'fields': ['user']}
+  {'collection': 'verifications', 'fields': ['user'], 'type': 'persistent'},
+  {'collection': 'verifications', 'fields': ['name'], 'type': 'persistent'},
+  {'collection': 'sponsorships', 'fields': ['expireDate'], 'type': 'ttl', 'expireAfter': 0}
 ]
```

Comment:
I notice that some indexes don't appear in this list.  Is there somewhere else they get created?

For example:  http://localhost:3434/_db/_system/_admin/aardvark/index.html#cIndices/groups

![image](https://user-images.githubusercontent.com/205792/103455430-fc396e80-4ca1-11eb-87a9-5fd1af6b361a.png)

Reply:
I actually don't see `founders[*]` being used anywhere in the node services, so that can probably be removed, unless you know where it might be used, but it seems like the index on `seed` must be useful somewhere.

I looked for other fields that are being filtered in queries, but didn't find any.  I didn't search all the python code though.  If a field is being filtered, it should have an index, unless it's a small collection.

Reply:
'seed' is being used by some of the anti-sybil algorithms, so it should have an index.